README.md

```diff
@@ -46,10 +49,11 @@ Or install it yourself as:
   client.close
 ```
 
-Alternately, you can pass a block to `.open` without the need to explicitly close the client
+Alternately, you can pass a block to `.open` without the need to explicitly close the client.
```

Comment:
should be `Alternatively`, not `Alternately`.  See https://www.grammar.com/alternately_vs._alternatively